code/modules/events/money_spam.dm

```diff
@@ -78,7 +78,7 @@
 					"We are happy to inform you that due to the delay, we have been instructed to IMMEDIATELY deposit all funds into your account",\
 					"Dear fund beneficiary, We have please to inform you that overdue funds payment has finally been approved and released for payment",\
 					"Due to my lack of agents I require an off-world financial account to immediately deposit the sum of 1 POINT FIVE MILLION credits.",\
-					"Greetings sir, I regretfully to inform you that as I lay dying here due to my lack ofheirs I have chosen you to recieve the full sum of my lifetime savings of 1.5 billion credits")
+					"Greetings sir, I regretfully to inform you that as I lay dying here due to my lack ofheirs I have chosen you to receive the full sum of my lifetime savings of 1.5 billion credits")
```

Comment:
If we go with the version that those are not intentional typos then its worth fixing "ofheirs" and the missing dot at the end too.

Reply:
i missed this, i think we should keep the poor spelling

Reply:
If we keep it then a comment would be good, because this is the second PR trying to fix it 😆